client/src/components/styles/Menu.scss

```diff
@@ -0,0 +1,69 @@
+@import "variables";
+@import "mixin";
+.navig {
+  @include position(0.5em,relative,60px);
+  @include flex_item(center,space-around);
+}
+.logo_component {
+  @include flex-item(center,space-around);
+  @include position(0 0.3em 0 0 ,static,28px);
+  .jsgram{
+    font-family: "Georgian";
+    font-weight: italic;
+    font-size: 26px;
+  }
+}
+
+.icons_list{
+  @include flex_item(normal,normal);
+  padding-left:1%;
+  .li_component{
+    display: inline;
+    list-style: none;
+  }
+}
+.center_part {
+  @include position(0,relative,59%);
+  @include flex_item(normal,normal);
+  align-content:center;
+}
+
+.input_input{
+  outline: 0;
+  transition: 3s;
+  &::placeholder{
+    @include input_text(0em,$color-grey);
+  }
+  &[type=search]
+  {
+    @include input_text(1.33rem,$color-black);
+ }
+}
+
+.center_part:hover {
+  .find_icon {
+    @include red_transition();
+  }
+  .input_input  {
+    border-radius: 20px;
+    transition: 3s;
+  }
+}
+.find_icon {
+  @include position(7px 4px 7px 7px,absolute,22px);
+  color: $color-grey;
+}
+.li_component:hover
+.logo_component:hover {
+  .inst_icon, .jsgram{
+    @include red_transition();
+}.center_part:hover {
```

Comment:
could u make this part of code more readable ?

Reply:
fixed